web/src/hooks/use-get-workflow-api.ts

```diff
@@ -29,5 +29,6 @@ export const useGetWorkflowApi = (workflowId: string, doApiCall: boolean) => {
 		queryKey: ["workflow", workflowId],
 		queryFn: () => getWorkflow({ workflowId }),
 		enabled: doApiCall,
+		gcTime: 0,
```

Comment:
Can you add a comment here that this disables the cache?

Reply:
or instead: let's introduce a constant `const DISABLE_CACHE = 0;` and replace that with the magic number here.